literature.bib

```diff
@@ -11999,13 +12002,13 @@ @inproceedings{JDB:SPLC07
 }
 
 @INPROCEEDINGS{FW:WMA07, 
-	author={Fraser, G. and Wotawa, F.}, 
-	title={{Mutant Minimization for Model-Checker Based Test-Case Generation}}, 
-	booktitle={in Proc. of Workshop on Mutation Analysis, Published with Proc. of Testing: Academic and Industrial Conference Practice and Research Techniques}, 
-	month=SEP, 
-	pages={161-168}, 
-	doi={10.1109/TAIC.PART.2007.30},
-	year=2007
+	author = {Fraser, G. and Wotawa, F.}, 
+	title = {{Mutant Minimization for Model-Checker Based Test-Case Generation}}, 
+	booktitle = {in Proc. of Workshop on Mutation Analysis, Published with Proc. of Testing: Academic and Industrial Conference Practice and Research Techniques}, 
```

Comment:
same here

Reply:
Again, not mine, just fixed spaces.

Reply:
https://github.com/tthuem/BibTags/blame/master/literature.bib#L11776